src/samples/ChildBenefitsClaim/index.tsx

```diff
@@ -572,6 +572,15 @@ export default function ChildBenefitsClaim() {
     } else if (authType && authType === 'gg-dev') {
       authService = 'GovGateway-Dev';
     }
+
+    // If the continer / case is opened then close the container on signout to prevent locking.
+    if (bShowPega) {
+      PCore.getContainerUtils().closeContainerItem(
+        PCore.getContainerUtils().getActiveContainerItemContext('app/primary'),
+        { skipDirtyCheck: true }
+      );
+    }
+
```

Comment:
Can I ask why please?

Currently if I were to try and close the container after logout is successful, the unlock call will fail because the container no longer exists.

Reply:
my only concern was if user click on signout button and due to some issue sigh out not happen but we are closing container.

Reply:
I've tried to use the invoke method but this doesn't work as the container needs to close to ensure the claim is unlocked.